pages/_app/index.jsx

```diff
@@ -21,12 +26,21 @@ class App extends NextApp {
     }
   }
 
-  store = null
-
-  componentDidMount() {
+  redirectToLogin = () => {
     const { router } = this.props
-    const { dataProvider, activity } = new Route(router.asPath)
-    this.store = initStore({ dataProvider, activity })
+    const pathname = '/login.html'
+    router.replace(pathname, pathname)
```

Comment:
[Somewhere in the doc](https://nextjs.org/docs/api-reference/next/router#routerpush) I read:
> You don't need to use _Router_ for external URLs, `window.location` is better suited for those cases.

Reply:
Interesting. Is `./login.html` considered as external url?

Reply:
It cannot be loaded by Next so it should be considered 'external', in my opinion.